pkg/process/net/common_windows.go

```diff
@@ -0,0 +1,30 @@
+// +build windows
+
+package net
+
+import (
+	"fmt"
+	"github.com/DataDog/datadog-agent/pkg/ebpf"
+)
+
+const (
+	statusURL      = "http://localhost:3333/status"
+	connectionsURL = "http://localhost:3333/connections"
+	statsURL       = "http://localhost:3333/debug/stats"
+	netType        = "tcp"
+)
+
+// SetSystemProbePath sets where the System probe is listening for connections
+// This needs to be called before GetRemoteSystemProbeUtil.
+func SetSystemProbePath(path string) {
+	globalSocketPath = path
+}
+
+// CheckPath is used in conjunction with calling the stats endpoint, since we are calling this
+// From the main agent and want to ensure the probe listener exists
+func CheckPath() error {
+	if globalSocketPath == "" {
+		return fmt.Errorf("remote tracer has no path defined")
+	}
+	return ebpf.ErrNotImplemented
```

Comment:
As this function returned always an error do we need https://github.com/DataDog/datadog-agent/pull/5179/files#diff-d59058de9a1cf992cafb3e206b8777b3R26-R28 ?

Do we need https://github.com/DataDog/datadog-agent/pull/5179/files#diff-d59058de9a1cf992cafb3e206b8777b3R19-R21 as we also defined `SetSystemProbePath` here https://github.com/DataDog/datadog-agent/pull/5179/files#diff-d59058de9a1cf992cafb3e206b8777b3R26-R27?

Reply:
Ahhh, these are good points. I'm going to move the SetSystemProbePath() function into the common file, and only return an error in CheckPath for the windows file, and I think this should address your comment.